recipe/build.sh

```diff
@@ -20,7 +20,9 @@ fi
 cmake ${CMAKE_ARGS} .. \
       -DCMAKE_BUILD_TYPE=Release \
       -DBUILD_PYTHON_INTERFACE=ON \
+      -DPython_EXECUTABLE=$PYTHON \
```

Comment:
```suggestion
      -DPython3_EXECUTABLE=$PYTHON \
```

Can we try this, just to see how many other issues we have ? :)